diagnostics-app/src/environments/environment.prod.ts

```diff
@@ -3,5 +3,6 @@
 // found in the LICENSE file.
 
 export const environment = {
-  production: true
+  production: true,
```

Comment:
why do you need this field in both files?

Reply:
`environment.prod` will be loaded when we build for production. But since we don't need this, I'll remove it for now.

Reply:
I mean the "production" field not the file.

Reply:
According to this: https://angular.io/guide/build#configure-environment-specific-defaults, only one object is loaded, therefore, we need to have this flag for production.